proof/refine/RISCV64/KHeap_R.thy

```diff
@@ -645,6 +958,29 @@ proof -
                   elim: cte_wp_atE' canonical_address_add)
 qed
 
+(* FIXME rt merge: move to Word_lib *)
+lemma max_word_minus_1[simp]: "0xFFFFFFFF + 2^x = (2^x - 1::32 word)"
+  by simp
```

Comment:
This lemma is unlikely to be useful in RISCV (should need 64 word). It would be nice to find a generic phrasing of this, but it would be Ok for now to just add a 64 bit version.